src/app/content/selectors.ts

```diff
@@ -52,7 +52,7 @@ export const loadingPage = createSelector(
 
 export const pageParam = createSelector(
   localState,
-  (state) => state.params ? state.params.page : null
+  (state) => state.params ? state.params.page.slug : null
```

Comment:
it looks like 

`findArchiveTreeNodeBySlug` and `getPageIdFromUrlParam` should be somehow combined or set up to share some logic that will find the page from the tree based on the page param.

we're trying to move away from the assumption that the page param is just a slug, so we shouldn't be introducing more places that rely on that